docs/tutorials/migrate-to-mui5.md

```diff
@@ -0,0 +1,34 @@
+---
+id: migrate-to-mui5
+title: Migrating from Material UI v4 to v5
+description: Additionally resources to the Material UI migration guide specifically for Backstage
+---
+
+Before diving into the specific changes in Backstage, it may be helpful to take a look at the [Migration Guide provided by MUI](https://mui.com/material-ui/migration/migration-v4/). It breaks down the differences between v4, v5 and will make it easier to understand the impact on your Backstage instance & plugins.
+
+To support Material UI v5 in addition to v4, we have introduced a `UnifiedThemeProvider`. This allows a gradual migration by running both versions in parallel. To use it, you need to update your app's `ThemeProvider` in `app/src/App.tsx` as follows:
```

Comment:
When do we expect these docs to go out? Should we mentioned you need to be on at least `1.15.0-next.2` or on the monthly `1.15.0` that comes out on the 20th?

Reply:
With `1.15.0` coming out tomorrow, you could just say to make sure you are on the latest version? I'm only mentioning this as I've seen on Discord people will try and migrate to new things but not do the Backstage upgrade and then have issues.